src/render/attrs/index.js

```diff
@@ -10,6 +10,42 @@ const attrsNotToRender = {
   dangerouslySetInnerHTML: true
 };
 
+const attrsWithExplicitBoolValue = {
+  "aria-expanded": true,
+  "aria-haspopup": true
+};
```

Comment:
I will try to make a quick POC this evening (being in Europe, this mean in the coming 6 hours) to explain it, as it is pretty hard to just describe it without code.